frontend/controller/actions/group.js

```diff
@@ -436,7 +436,7 @@ export default (sbp('sbp/selectors/register', {
           if (!proposal.votes[getters.ourUsername]) {
             await sbp('gi.actions/group/proposalVote', {
               contractID: groupID,
-              data: { proposalHash, vote: VOTE_FOR, passPayload: { secret: '' } },
+              data: { proposalHash, vote: VOTE_FOR, passPayload: { secret: '', automated: true } },
```

Comment:
`automated` doesn't need to be here, you have it correct below. Please revert the change above.